content/en/docs/Creating derivatives/building_gce_images.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Build GCE images"
+linkTitle: "Build GCE images"
+weight: 4
+date: 2017-01-05
+description: >
+  This section documents the procedure to deploy cOS images
+  in GCE public cloud provider from scratch.
+---
+
+
+Requirements:
+
+* cOS-toolkit source
+* GCE access keys with the appropriate roles and permissions
+* [gsutil](https://cloud.google.com/storage/docs/gsutil) and [gcloud](https://cloud.google.com/sdk) tools
+
+The suggested approach high level view is building cOS packages and generating a RAW image from
+them. That would allow us to transform that RAW image in a valid GCE blob that can be transformed into a VM image ready
+to be launched.
+
+This generates a `vanilla` image that boots into [recovery mode](../../getting-started/recovery) and can be used to deploy
+whatever image you want to the VM. Then you can snapshot that VM into a VM image ready to deploy with the default cOS
+system or your derivative.
+
+## Building the packages
+
+We need to have the packages built locally in order to generate a proper RAW image. Just run:
+
+```bash
+sudo make build
```

Comment:
Maybe this should go under `dev/` as it is documenting how to build `cOS` and not a derivative raw disk

Reply:
This is already documented under development and linked 2 lines below, but I dont want the reader to keep jumping pages. 

If you want to know more about building, there is your link to extend your info about the whole shebang, otherwise you are here to generate an gce image, this are the straightforward instructions all contained in the same page.

Reply:
the "build" aspect imho shouldn't be user-facing in that way. That's where we should draw a line between different personas interacting with the docs. Building cOS should be optional for the reader, unless they go to the "dev" way which gives more information on how to build everything from scratch. When we will have releases and downloadable artifacts, getting a vanilla or recovery to _bootstrap_ a customization is way more convenient than asking the user to build everything by himself locally

Reply:
Probably we could omit this section and assume the user has only to download the artifact from github. Currently we have it only as artifacts of runs, which is kind of cumbersome to point them there. But as long as we start running releases at github I'd say we should include the ISO and the Vanilla image (we could argue to what extend is needed to release three of them, as then are basically the same).

Reply:
I was struggling to add both methods, but pointing to our current setup for getting the artifacts from github is a bit crappy. I think this is ok **now** but needs change once we publish the artifacts as releases as this means users dont need to build anything, just download the recovery/vanilla image and upload. Then this part can be moved to `development` as it would be the only user case for building it locally.

Reply:
> include the ISO and the Vanilla image (we could argue to what extend is needed to release three of them, as then are basically the same).

To be fair, I would release all 3. That way you get the image that you need directly, no need to mess around with modifying it, just download update, create image, done. It really costs us nothing but a bit more of time to publish all 3 (especially if we only publish on release) but we gain a lot of good faith by providing everything ready to roll.

On master/nigthly/pr builds I would not do the same of course, that make no sense as those are for dev only